intro.js

```diff
@@ -536,10 +543,22 @@
         currentTooltipPosition = _determineAutoPosition.call(this, targetElement, tooltipLayer, currentTooltipPosition);
       }
     }
+
     targetOffset  = _getOffset(targetElement);
     tooltipOffset = _getOffset(tooltipLayer);
     windowSize    = _getWinSize();
 
+    if (typeof(helperNumberLayer) != 'undefined' && helperNumberLayer != null) {
+      var isFullScreenTarget = windowSize.width - targetOffset.width < 40;
+      helperNumberLayer.style.top  = null;
+      helperNumberLayer.style.left = isFullScreenTarget ? -targetOffset.left + 5 + 'px' : null;
+
+      const vertical = targetOffset.top > 10 ? 'top' : 'bottom';
+      const horizontal = targetOffset.left > 10 ? 'left' : 'right';
```

Comment:
Can we use `var` here so as to keep better browser compatibility with this repo, I realize it will probably be ran through babel further down the line, but I'd rather keep it consistent throughout this file.